frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -45,11 +45,11 @@ function getDesignSystemCssVariables(theme: MantineTheme) {
     --mb-color-bg-dark: ${theme.fn.themeColor("bg-dark")};
     --mb-color-brand: ${theme.fn.themeColor("brand")};
 
-    --mb-color-brand-light: color-mix(in srgb, var(--mb-color-brand) 53%, #fff);
+    --mb-color-brand-light: color-mix(in srgb, var(--mb-color-brand), #fff 80%);
```

Comment:
These colors are eye-balled, but they passed loki tests compared to the colors from `palette.ts`